app/models/image.rb

```diff
@@ -5,6 +5,10 @@ class Image < ActiveRecord::Base
   has_many :tags, through: :image_tags
   has_many :comments
   
+  has_many :likes, dependent: :destroy
+  has_many :user_likes, class_name: "Likes",
+                        foreign_key: "image_id"
```

Comment:
I believe the `class_name:` value should be the singular "Like" to match the name of the model class (but not sure about that)